### DISCUSSION TITLE
Using KEDA only for scale-to-zero and disable HPA

### DISCUSSION THREAD

**Discussion by hholst80** [kedacore/keda#1690] [Q&A / Need Help]
We have not implemented a autoscaling feature and we would like to enable only a scale-to-zero feature as of now.  

Is that possible, and if so, what is the recommended way of doing it with KEDA?

thanks!

**Comment by tomkerkhove** [comment]
This is definately supported! You can define `maxReplicaCount: 1` and it will not scale beyond that.

See https://keda.sh/docs/2.2/concepts/scaling-deployments/#scaledobject-spec for more detailed information
